src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -328,7 +329,8 @@ export default {
 				variables: {
```

Comment:
Please pull the creation of the "variables" values up before `this.apollo.query` so you can conditionally add in the teams sortBy param IF `this.displayType === 'teams'`.